build.gradle

```diff
@@ -0,0 +1,158 @@
+plugins {
```

Comment:
I think it would be useful to indent this file like the source code and resources.
The source code is indented with 2 spaces, brackets have their own lines and empty lines between large blocks.